code/index.html

```diff
@@ -2,21 +2,119 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="stylesheet" href="font-awesome-4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous" />
   </head>
+  
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <div class="logo">
+        <img src="./logo.png" alt="" srcset="">
+        <p class="logo-text">Travelist Magazine</p> 
+      </div>
+
+      <!-- navbar content -->
+
+      <input type="checkbox" id="nav-toggle" class="nav-toggle">
+      <nav class="nav">
+        <ul>
+          <li><a class="nav-item" href="#">Newsletter</a></li>
+          <li><a class="nav-item" href="#">Subscribe</a></li>
+          <li><a class="nav-item" href="#">Sign in</a></li>
+          <li><a class="nav-item" href="#"><i class="fa fa-search" aria-hidden="true"></i></a></li>
+        </ul>
```

Comment:
I like the comment <!--navbar content--> so it´s clear on where we are in the code. 

I would suggest to push in the "hidden="true" so the code looks even and clean :)